app/components/chart-donut.js

```diff
@@ -43,6 +43,14 @@ export default Component.extend({
       return;
     }
 
+    if (!isIcon) {
+      svg.classed("svg-container", true)
+      .attr("preserveAspectRatio", "xMinYMin meet")
+      .attr("viewBox","0 0 " + width + " " + height)
+      //class to make it responsive
+      .classed("svg-content-responsive", true);
```

Comment:
We could or "should" make all SVG responsive, with the exception of the Icons of course.  We could add a parent container to the SVG as I mentioned in the comment above this one.